includes/cli/class-amp-cli-validation-command.php

```diff
@@ -267,6 +194,129 @@ public function run( $args, $assoc_args ) {
 		WP_CLI::line( sprintf( 'For more details, please see: %s', $url_more_details ) );
 	}
 
+	/**
+	 * Provides the associative args for the command.
+	 *
+	 * @return array
+	 */
+	private function get_assoc_args() {
+		if ( ! is_array( $this->assoc_args ) ) {
+			$this->assoc_args = [];
+		}
+
+		return array_merge(
+			[
+				self::LIMIT_URLS_ARGUMENT        => 100,
+				self::INCLUDE_ARGUMENT           => [],
+				self::FLAG_NAME_FORCE_VALIDATION => false,
+			],
+			$this->assoc_args
+		);
+	}
+
+	/**
+	 * Provides the ValidationURLProvider instance.
+	 *
+	 * @return ValidationURLProvider
+	 */
+	private function get_validation_url_provider() {
+		if ( ! is_null( $this->validation_url_provider ) ) {
+			return $this->validation_url_provider;
+		}
+
+		$assoc_args = $this->get_assoc_args();
+
+		$include_conditionals      = [];
+		$force_crawl_urls          = false;
+		$limit_type_validate_count = (int) $assoc_args[ self::LIMIT_URLS_ARGUMENT ];
+
+		/*
+		 * Handle the argument and flag passed to the command: --include and --force.
+		 * If the self::INCLUDE_ARGUMENT is present, force crawling or URLs.
+		 * The WP-CLI command should indicate which templates are crawled, not the /wp-admin options.
+		 */
+		if ( ! empty( $assoc_args[ self::INCLUDE_ARGUMENT ] ) ) {
+			$include_conditionals = explode( ',', $assoc_args[ self::INCLUDE_ARGUMENT ] );
+			$force_crawl_urls     = true;
+		} elseif ( isset( $assoc_args[ self::FLAG_NAME_FORCE_VALIDATION ] ) ) {
+			$force_crawl_urls = true;
+		}
```

Comment:
This is already taken care of above.
```suggestion
		}
```